cognite/client/data_classes/workflows.py

```diff
@@ -32,6 +32,8 @@
     "skipped",
 ]
 
+WorkflowStatus: TypeAlias = Literal["completed", "failed", "running", "terminated", "timed_out"]
```

Comment:
"paused" has been removed from the possible statuses. 
https://github.com/cognitedata/service-contracts/pull/2535